lib/blpapi-wrapper.ts

```diff
@@ -257,9 +257,8 @@ export class Session extends events.EventEmitter {
     private doRequest(uri: string,
                       requestName: string,
                       request: any,
-                      callback: IRequestCallback,
-                      isAuthRequest: boolean,
-                      identity?: blpapi.Identity): void
+                      perform: (correlatorId: number) =>  void,
```

Comment:
`perform` is a bit ambiguous.  Maybe it is that `doRequest` isn't really doing the request, `perform` is.  I am trying to think of a clearer nomenclature.